apps/web/src/service/api/establishmentApi.ts

```diff
@@ -17,4 +17,17 @@ export default class EstablishmentApi extends RequestApi {
       return Result.err(error as Error)
     }
   }
+
+  async searchActivities(query: string): Promise<Result<CompanyActivityType[], Error>> {
+    const urlWithParams = `${this.url}searchNAF?searchTerm=${encodeURIComponent(query)}`
```

Comment:
TODO : Il faudrait éviter de se retrouver avec un nom de variable en dur. 
Sachant qu'il n'y a qu'un seul paramère, je me demande si c'est possible d'avoir 
const urlWithParams = `${this.url}searchNAF?${encodeURIComponent(query)}` 
et dans le backend 
 @Get('/searchNAF?{query}')
  public getOne(
    @Path() query: string,
  

Sinon il faudrait créer une interface dans la librarie common.